documentation/src/main/asciidoc/topics/ref_persistence_spi.adoc

```diff
@@ -0,0 +1,48 @@
+= Persistence SPI Classes
+
+The following class diagram shows the main interfaces of the persistence SPI:
+
+image::{images_dir}/Figure2_1_persistence_API.png[align="center", title="Persistence SPI"]
```

Comment:
This image is pretty out of date now :cry:

Reply:
yeah, let's kill it for now. I'm not sure we really want to maintain class diagrams like this for each release.